arangod/RocksDBEngine/RocksDBRestReplicationHandler.cpp

```diff
@@ -407,15 +408,25 @@ void RocksDBRestReplicationHandler::handleCommandInventory() {
 }
 
 /// @brief produce list of keys for a specific collection
+/// If the call is made with option quick=true, and more than
+/// 1 million documents are counted for keys, we'll return only
+/// the document count, else, we proceed to deliver the keys.
 void RocksDBRestReplicationHandler::handleCommandCreateKeys() {
   std::string const& collection = _request->value("collection");
   if (collection.empty()) {
     generateError(rest::ResponseCode::BAD, TRI_ERROR_HTTP_BAD_PARAMETER,
                   "invalid collection parameter");
     return;
   }
+
+  std::string const& quick = _request->value("quick");
+  if (!quick.empty() && !(quick == "true" || quick == "false")) {
```

Comment:
The `quick` seems to be assigned and validated only. But I can't see any code that would use it for anything else. Especially I don't see any code that would return the collection's count value if `quick` is set.

Reply:
It is now used further below.